src/handlers/motions/motionStaked/helpers.ts

```diff
@@ -0,0 +1,209 @@
+import { BigNumber } from 'ethers';
+import { MotionQuery, MotionStakes, UserStakes } from '~types';
+import { verbose } from '~utils';
+import { getMotionSide } from '../helpers';
+
+export const getRequiredStake = (
+  skillRep: BigNumber,
+  totalStakeFraction: BigNumber,
+): BigNumber =>
+  skillRep.mul(totalStakeFraction).div(BigNumber.from(10).pow(18));
```

Comment:
Tokens can have a different number of decimals, rather than the standard 18 as we have for most tokens